javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+var $countdownDisplay = document.querySelector('.countdown-display');
+
+function countdownDisplay() {
+  if ($countdownDisplay.textContent === '4') {
+    $countdownDisplay.textContent = '3';
+  } else if ($countdownDisplay.textContent === '3') {
+    $countdownDisplay.textContent = '2';
+  } else if ($countdownDisplay.textContent === '2') {
+    $countdownDisplay.textContent = '1';
+  } else if ($countdownDisplay.textContent === '1') {
```

Comment:
What if you wanted to start the countdown from `1000`? You should try to find a way to do this without needing a conditional block for each number.

Reply:
Fixed it! I added a variable to store the current count number (this was parsed from a string to a number)